### PR TITLE
Add ttl option to user documents creation and state to detailed screen

### PR DESCRIPTION
**Pull request by annakocheshkova**
* [ ] Has `CHANGELOG.md` been updated?
* [X] Are tests passing locally?
* [X] Are the files formatted correctly?
* [ ] Did you add unit tests?
* [X] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?
* [x] Did you check UI tests on the sample app? They are not executed on CI.

## Description
Adds State to detailed view (and also rewritten detailed view)
image
Adds Time to live option on New User Document screen:
image

### CODE REVIEW

File: apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/AppDocumentListAdapter.java [hunk 1]
```diff
@@ -28,13 +28,14 @@
     @NonNull
     @Override
     public AppDocumentListHolder onCreateViewHolder(@NonNull ViewGroup viewGroup, int i) {
-        return new AppDocumentListHolder(LayoutInflater.from(mContext).inflate(R.layout.item_view_app, null, false));
+        return new AppDocumentListHolder(LayoutInflater.from(mContext).inflate(R.layout.item_view_app, viewGroup, false));
     }
 
     @Override
-    public void onBindViewHolder(@NonNull AppDocumentListHolder holder, final int position) {
+    public void onBindViewHolder(@NonNull AppDocumentListHolder holder, @SuppressLint("RecyclerView") final int position) {
```
Comment (guperrot) [inline apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/AppDocumentListAdapter.java]: what is that warning you suppressed?

Comment (annakocheshkova) [reply apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/AppDocumentListAdapter.java]: Fair point. I copypasted it from anotehr plae for no good reason.

File: apps/sasquatch/src/projectDependency/java/com/microsoft/appcenter/sasquatch/activities/NewUserDocumentActivity.java [hunk 2]
```diff
@@ -33,14 +38,47 @@
...
+            @Override
+            public void onNothingSelected(AdapterView<?> parent) {
+            }
+        });
         addProperty();
     }
 
+    private void updateWriteOptions(int position) {
+        switch (position) {
+            case 0:
```
Comment (guperrot) [inline apps/sasquatch/src/projectDependency/java/com/microsoft/appcenter/sasquatch/activities/NewUserDocumentActivity.java]: use an enum

Comment (guperrot) [reply apps/sasquatch/src/projectDependency/java/com/microsoft/appcenter/sasquatch/activities/NewUserDocumentActivity.java]: Fixed

**Pull request merged**